src/content/docs/change-tracking/ci-cd/change-tracking-jenkins.mdx

```diff
@@ -147,14 +147,20 @@ The plugin we've created takes the GUID as a parameter. You can identify the GUI
       alt="Screenshot showing how to find Deployment Notifications"
       src={trackingJenkinsDeployNotification}
     />
-4. Complete the following fields:
+4. Complete the following fields (note REST API fields are also listed in case you're not using GraphQL):
```

Comment:
I don't the "note" is necessary here.

Maybe something more like "we've included the REST API fields here for people not using GraphQL"?